crates/api_crud/src/user/my_user.rs

```diff
@@ -23,16 +23,24 @@ pub async fn get_my_user(
   let local_user_id = local_user_view.local_user.id;
   let pool = &mut context.pool();
 
-  let (follows, community_blocks, instance_blocks, person_blocks, moderates, discussion_languages) =
-    lemmy_db_schema::try_join_with_pool!(pool => (
-      |pool| CommunityFollowerView::for_person(pool, person_id),
-      |pool| CommunityActions::read_blocks_for_person(pool, person_id),
-      |pool| InstanceActions::read_blocks_for_person(pool, person_id),
-      |pool| PersonActions::read_blocks_for_person(pool, person_id),
-      |pool| CommunityModeratorView::for_person(pool, person_id, Some(&local_user_view.local_user)),
-      |pool| LocalUserLanguage::read(pool, local_user_id)
-    ))
-    .with_lemmy_type(LemmyErrorType::SystemErrLogin)?;
+  // TODO this try join isn't working with LemmyError, just read them
+  // let (follows, community_blocks, instance_blocks, person_blocks, moderates,
+  // discussion_languages) =   lemmy_db_schema::try_join_with_pool!(pool => (
+  //     |pool| CommunityFollowerView::for_person(pool, person_id),
+  //     |pool| CommunityActions::read_blocks_for_person(pool, person_id),
+  //     |pool| InstanceActions::read_blocks_for_person(pool, person_id),
+  //     |pool| PersonActions::read_blocks_for_person(pool, person_id),
+  //     |pool| CommunityModeratorView::for_person(pool, person_id, Some(&local_user_view.local_user)),
+  //     |pool| LocalUserLanguage::read(pool, local_user_id)
+  //   ))
```

Comment:
Do all of these functions return the same error type?

Reply:
No, but I got it fixed now.